bitnami/aspnet-core/README.md

```diff
@@ -305,7 +305,7 @@ For example, you can deploy a sample [Kestrel server](https://docs.microsoft.com
 appFromExternalRepo.enabled=true
 appFromExternalRepo.clone.repository=https://github.com/dotnet/AspNetCore.Docs.git
 appFromExternalRepo.clone.revision=main
-appFromExternalRepo.publish.aspnetcore/fundamentals/servers/kestrel/samples/3.x/KestrelSample
+appFromExternalRepo.publish.aspnetcore/fundamentals/servers/kestrel/samples/6.x/KestrelSample
```

Comment:
Isn't this line missing `subFolder=` ? And hasn't this value been updated to `aspnetcore/performance/caching/output/samples/7.x/`?